pkg/controller/acmeorders/sync_test.go

```diff
@@ -39,7 +39,7 @@ import (
 	"github.com/jetstack/cert-manager/test/unit/gen"
 )
 
-func TestSyncHappyPath(t *testing.T) {
+func TestSync(t *testing.T) {
```

Comment:
I think these were kept separate originally so we didn't have failing/bad state test cases mixed in with the 'happy' (working) cases - largely to avoid this becoming a big umbrella test case and to encourage some splitting up of different areas/tests

Reply:
I probably find it easier to be able to view test cases related to one part of code together, I don't find the splitting of happy/not happy path that useful especially in case of a reconciler as it's not that clear what the split is.